.github/workflows/test-fast.yml

```diff
@@ -75,10 +75,6 @@ jobs:
 
     runs-on: ${{ matrix.platform }}
 
-    # [WINDOWS-CI] - Find other instances of this string to find related snippets when this issue is
-    # resolved: https://github.com/pulumi/pulumi/issues/8820
-    continue-on-error: ${{ matrix.platform == 'windows-latest' }}
```

Comment:
This is removed since it didn't work - somehow the "lost worker" condition is not the same as an error, and this clause has no effect on what it's trying to address.